### PR TITLE
Inspect an initial version of the public sample data

### PR DESCRIPTION
**Pull request by zmbc**
Code that generated this from a 20k population sim run: https://github.com/ihmeuw/vivarium_census_prl_synth_pop/pull/251

### CODE REVIEW

File: model_validation/2023_03_20_initial_public_sample_data.ipynb [hunk 1]
```diff
@@ -0,0 +1,1087 @@
+{
```
Comment (aflaxman) [inline model_validation/2023_03_20_initial_public_sample_data.ipynb:1]: <p>Uh oh!  We better fix this if so.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research_prl/pull/30/discussion/#comment-1142626233'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

**Review by aflaxman — APPROVED** [review]

File: model_validation/2023_03_20_initial_public_sample_data.ipynb (hunk 1 above)
Comment (zmbc) [reply model_validation/2023_03_20_initial_public_sample_data.ipynb:1]: I think this is not a bug, but a choice to leave this part of the processing to pseudopeople -- but we should confirm that this work isn't being forgotten.

**Comment by aflaxman** [comment]
Oh, why?

**Comment by zmbc** [comment]
@aflaxman Why leave the processing to pseudopeople? I'm not sure.

**Comment by aflaxman** [comment]
Maybe this is because if you worked in 2019 and died that year, you still get a W2.  But it seems like there could be some bug in this.  Can you bounce this into slack, so @Caroline ***@***.***> can help us keep track of it?

--Abie

File: model_validation/2023_03_20_initial_public_sample_data.ipynb (hunk 1 above)
Comment (NathanielBlairStahn) [inline model_validation/2023_03_20_initial_public_sample_data.ipynb:1]: <p>I'm just trying to understand what this is saying. So the <code>not_in_census</code> dataframe contains people who show up in 2029 taxes (1040s) but but do not show up in the 2030 census, correct? How should we interpret the dates you calculated? Does the date represent either 1) the approximate date at which the simulant was recorded by the 1040 observer, if that simulant was alive at that time, or 2) the time of death of the simulant if they died before the 1040 observer recorded them?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research_prl/pull/30/discussion/#comment-1142739427'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

**Comment by NathanielBlairStahn** [comment]
> @aflaxman Why leave the processing to pseudopeople? I'm not sure.

Maybe it's for the "improper inclusion" noise (which I think we deprioritized implementing for now), i.e., having some number of people show up in tax data even if they're not supposed to because they're dead or don't make enough money.

File: model_validation/2023_03_20_initial_public_sample_data.ipynb (hunk 1 above)
Comment (zmbc) [reply model_validation/2023_03_20_initial_public_sample_data.ipynb:1]: Yep that's exactly it! The fact that many of the dates are before 2030 indicates that many of the extra people are no longer alive.

**Comment by zmbc** [comment]
To be clear, people who have died are allowed to file taxes (well, someone else is supposed to file taxes for them). It's only the income threshold that needs to be added in pseudopeople -- which will inherently filter out anyone who has been dead for multiple years.

**Review by NathanielBlairStahn — APPROVED** [review]

**Pull request merged**